front/app/modules/commercial/content_builder/admin/components/contentBuilderToggle.tsx

```diff
@@ -1,12 +1,83 @@
-import React from 'react';
+import QuillMultilocWithLocaleSwitcher from 'components/UI/QuillEditor/QuillMultilocWithLocaleSwitcher';
+import React, { useState } from 'react';
```

Comment:
You can put the React import on top and group the rest with comments - components, styles, etc. It's not a hard rule, but a convention we have in the codebase.